worldedit-core/src/main/java/com/sk89q/worldedit/regions/NullRegion.java

```diff
@@ -74,17 +75,17 @@ public int getLength() {
 
     @Override
     public void expand(BlockVector3... changes) throws RegionOperationException {
-        throw new RegionOperationException("Cannot change NullRegion");
+        throw new RegionOperationException(TranslatableComponent.of("worldedit.selection.null.error.immutable"));
```

Comment:
Why is this categorized under `selection`? This technically has nothing to do with a selection, a plugin can expand a region without a corresponding selection being changed.

Reply:
I felt it made more sense to categorise it alongside the other region operation messages